components/structured-data.tsx

```
@@ -0,0 +1,109 @@
+      url: "https://easyshifthq.com",
+      logo: {
+        "@type": "ImageObject",
+        url: "https://easyshifthq.com/favicon.svg",
+      },
+    },
+  }
+
+  return (
+    <script
+      type="application/ld+json"
+      dangerouslySetInnerHTML={{ __html: JSON.stringify(schema) }}
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)